ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -280,7 +281,12 @@ class GraphsContainer extends React.Component {
           }`}
           ref={this.tooltip}
         >
-          <span className="close mr-3 my-2 ml-2" onClick={this.closeTooltip}>
+          <span
```

Comment:
Since this acts as a button it'd be a good idea to change it to a button element since it's better to use elements with semantic meaning. We do use `react-strap` and using a `Button` component is a good option that I think eliminates the need to add `role="button"`.

Reply:
ok, changed to `<Button>`. Also dropped the `tabIndex={}` attribute, since it's an interactive element.